### DISCUSSION TITLE
英文摘要的间距也太大了吧，请问这是要求中规定的嘛？

### DISCUSSION THREAD

**Discussion by bakulaibuji** [sjtug/SJTUThesis#871] [提问] [answered]
如果要调的话，具体该怎么调呢？

**Comment by LogCreative** [comment]
学位类型是什么，以及是哪里的间距

**Reply by bakulaibuji** [reply]
硕士论文，是英文摘要的间距，感觉好大~

**Comment by LogCreative** [comment, marked as answer]
可以参照 #862 的讨论，如果想减少英文摘要的正文间距，可以使用
```latex
\begin{abstract*}
  \linespread{1.2}\selectfont
  % English abstract ...
\end{abstract*}
```

[《撰写指南》](https://www.gs.sjtu.edu.cn/info/1143/5801.htm)并没有对行间距做特殊说明，意味着可以灵活处理，Word 版对英文间距的规定是行距 20 磅。

现在英文摘要的间距为 1.56 倍文字高度，在硕士学位论文中默认字号为小四号，即 12 磅文字大小，那么现在的行间距长度为
$$12\text{ bp}\times 1.56 = 18.72\text{ bp}$$
这个长度甚至略小于 20 磅，当然也会随着不同的文档情形做自动的细微调整。所以一般不用做什么调整。
